spec/features/admin_company_value_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'rails_helper'
+# frozen_string_literal: true
+RSpec.describe 'Admin employees spec', type: :feature do
+  let!(:admin) { create(:admin) }
+  let!(:company_value) { build(:company_value) }
+
+  it 'CRUD company value' do
+    visit admins_root_path
+
+    fill_in 'admin[email]', with: admin.email
+    fill_in 'admin[password]', with: admin.password
+    click_button 'Log in'
```

Comment:
Consider using devise login helper here - you have already covered user logging with tests, so there is not necessary to check it here again.

https://localcoder.org/how-to-access-devise-current-user-in-a-rspec-feature-test